include/small/abstract_layer.hpp

```diff
@@ -303,7 +309,7 @@ void inline kernel(
             for (uint32_t ii = 0; ii < _F_cb / _UNROLL; ii++)
             {
                 /// @note using platform C_ob
```

Comment:
With the change from FLOAT_C_ob to _C_ob, this comment is no longer correct. Remove?